index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I have noticed there are no linters I suggest you to  install the linters, indeed the linters are efficient ways to flag all the typos and syntax errors. That will help you to save a lot of time while doing the debugging of your code.
please follow the link below to learn about Linters installation:
 **https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md**

Reply:
- [ ] [REQUIRED CHANGES]
Hi @silviatofana, you've made an impressive review and a great one for a start, however there below are some suggestions that could make it better 

:point_down: 

![review](https://user-images.githubusercontent.com/59397159/165143188-3b281740-9f66-4ac3-aa35-e7c6064ec25a.png)

- According to your review, you have told the student to follow microverse readme template, but you've not pointed out the headers that were omitted by the student. As the student doesn't know exactly what to do regarding your review. 
Kindly, tell them what is missing so they know exactly what to add to make their readme more professional

Reply:
![clear](https://user-images.githubusercontent.com/59397159/165144686-858b61f4-05ee-4bf0-8cdd-6212431d9806.png)

:point_up_2: 
- [ ] Also, in addition to your review regarding the `<meta http-equiv="X-UA-Compatible" content="IE=edge">`, It is highly recommended to add it to your HTML file because the `X-UA-compatible meta tag` allows you to choose the version on internet explorer your website should be rendered as. There are lots of reasons to why you should add it. check this [answers from stackOverflow](https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do) to get more clarification.

Reply:
Please in README file follow the steps request to complete it in a professional way 
- project title
- languages use
- contact form from you LinkedIn, GitHub
- programs install
- prerequisites
- -usage
- authors
- contributing